crates/feature_flags/src/feature_flags.rs

```diff
@@ -33,6 +38,10 @@ pub trait FeatureFlag {
     fn enabled_for_staff() -> bool {
         true
     }
+
+    fn enabled_in_development() -> bool {
+        Self::enabled_for_staff()
```

Comment:
Doesn't this mean that in development everyone will see staff-flagged features, not just Zed staff members?

Reply:
yes, but I figured that was ok. If you are building your own zed, you can always sneak in here and set is staff to true

Reply:
I guess it's the difference between having to go out of your way to do it (if someone is itching to play with something we're working on) vs having every staff-shipped feature immediately available to anyone working on Zed.

This would also remove the ability to use `ZED_DISABLE_STAFF=1` in development to view Zed as the perspective of a non-staff user, which I use pretty heavily.

Reply:
So I'm not sure I agree with the premise of "running development mode == enable all of the feature flags".

Reply:
`ZED_DISABLE_STAFF` should work, that was an oversight on my part.

In development, I'm usually not signed in, and when I am I  don't want to wait on the network to test features I'm working on.